lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs

```diff
@@ -148,6 +148,65 @@
 
     {{/accordion-list-item}}
     {{!-- edit cluster --}}
+    {{#accordion-list-item title=(t "clusterNew.ociocne.advanced.title")
+                           detail=(t "clusterNew.ociocne.advanced.detail")
+                           expandAll=expandAll
+                           expand=(action expandFn)
+                           expandOnInit=true}}
+      {{#accordion-list-item title=(t "clusterNew.ociocne.addons.yaml.title")
+                             detail=(t "clusterNew.ociocne.addons.yaml.detail")
+                             expandAll=expandAll
+                             expand=(action expandFn)
+                             expandOnInit=false}}
+        <div class="row">
+          <button
+            class="btn btn-sm bg-primary inline-block mb-5"
+            style="margin-top: -5px;" type="button" {{action "addYAML"}}
+          >
+            {{t "clusterNew.ociocne.addons.yaml.addNew"}}
+          </button>
+        </div>
+        {{#each yamls as |yaml|}}
+          <div class="row">
+            <div class="col span-4">
+              {{input-yaml
+                showUpload=true
+                showDownload=true
+                canChangeName=false
+                value=yaml.body
+                autoResize=true
+                loading=loading
+              }}
+            </div>
+          </div>
+
+          <button
+            class="btn btn-sm bg-primary inline-block mb-5"
+            style="margin-top: -5px;" type="button" {{action "deleteYAML" yaml}}
+          >
+            {{t "clusterNew.ociocne.addons.yaml.delete"}}
+          </button>
+        {{/each}}
+      {{/accordion-list-item}}
+      {{#accordion-list-item title=(t "clusterNew.ociocne.addons.title")
+                             detail=(t "clusterNew.ociocne.addons.detail")
+                             expandAll=expandAll
+                             expand=(action expandFn)
+                             expandOnInit=false}}
+        <p class="text-bold">{{t 'clusterNew.ociocne.addons.registry.heading'}}</p>
```

Comment:
Do we want to keep the container registry section? Also wasn't mentioned in the doc

Reply:
yeah its needed for Verrazzano installation